tests/src/ondevice_tests/crashdump_test.cpp

```
@@ -0,0 +1,394 @@
+#include <vector>
+
+#include "depthai/depthai.hpp"
+#include "utility/Environment.hpp"
+
+namespace {
+
+using namespace std::chrono_literals;
+namespace fs = std::filesystem;
+
+constexpr auto NO_CALLBACK_TIMEOUT = 5s;
+constexpr auto CRASH_DUMP_CALLBACK_TIMEOUT = 30s;
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)